states/pause.lua

```diff
@@ -19,6 +19,11 @@ function pause.new(savedGame, isMenu)
 		{x = ARENA_WIDTH / 2 - 200, y = 300, width = 400, height = 28, text = "Exit to menu", onPress = function() self:exit() end},
 		{x = ARENA_WIDTH / 2 - 200, y = 350, width = 400, height = 28, text = "Quit game", onPress = function() quitGame() end},
 	}
+	self.pauseButtonsHTML = {
+		{x = ARENA_WIDTH / 2 - 200, y = 200, width = 400, height = 28, text = "Continue", onPress = function() self:resume() end},
+		{x = ARENA_WIDTH / 2 - 200, y = 250, width = 400, height = 28, text = "Options", onPress = function() self.buttons = self.optionButtons end},
+		{x = ARENA_WIDTH / 2 - 200, y = 300, width = 400, height = 28, text = "Exit to menu", onPress = function() self:exit() end},
+	}
```

Comment:
Rather than keep track of a totally seperate list, lets just make it not spawn the last button if IS_HTML is true. So, basically you could keep that _second_ list as `self.pauseButtons`, then say
```lua
if not IS_HTML then
    self.pauseButtons[4] = { ... }
end
```
That way, if we want to change the buttons, we don't need to do it twice